### PR TITLE
Fix kubectl describe ingress annotations not sorted

### PR DESCRIPTION
**Pull request by zhouya0**
**What type of PR is this?**
> /kind bug
> /kind cleanup

**What this PR does / why we need it**:
Using `kubectl describe ingress` will see the `annotations` keeps changing because it didn't sort the annotations.
This origin PR was https://github.com/kubernetes/kubernetes/pull/16232. I think at that time he didn't think about using this `printAnnotationsMultilineWithIndent` function.
**Which issue(s) this PR fixes**:

Fixes # https://github.com/kubernetes/kubectl/issues/802

**Does this PR introduce a user-facing change?**:

```release-note
Fix describe ingress annotations not sorted.
```

### CODE REVIEW

**Comment by zhouya0** [comment]
/priority backlog
/kind bug
/assign @soltysh 

This is actually the same PR with https://github.com/kubernetes/kubernetes/pull/87275. But because of some code conflicts, I submitted multiple commits and hard to squash them together. So I closed that PR and reopen this.

**Comment by apelisse** [comment]
Looks good to me, but we need to agree with @soltysh :-)

**Comment by soltysh** [comment]
> Looks good to me, but we need to agree with @soltysh :-)

Like I said here: https://github.com/kubernetes/kubernetes/pull/87275#discussion_r382787237, I'd rather keep this PR simple, iow. w/o that error handling. 

After we agree on the right approach how to proceed with partial errors, I've already added that topic to next sig-cli call upcoming Wed, we can continue from there in a separate PR.

File: staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go [hunk 1]
```diff
@@ -2350,8 +2350,14 @@ func (i *IngressDescriber) Describe(namespace, name string, describerSettings de
...
-	endpoints, _ := i.CoreV1().Endpoints(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
-	service, _ := i.CoreV1().Services(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
+	endpoints, err := i.CoreV1().Endpoints(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Sprintf("<none> (failed to fetch endpoints: %v)", err)
+	}
+	service, err := i.CoreV1().Services(ns).Get(context.TODO(), backend.ServiceName, metav1.GetOptions{})
+	if err != nil {
+		return fmt.Sprintf("<none> (failed to fetch services: %v)", err)
+	}
```
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go]: Revert this change to previous state.

**Review by soltysh — CHANGES_REQUESTED** [review]

File: staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go (hunk 1 above)
Comment (zhouya0) [reply staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go]: Done :). (Can't pass the test case without this, so I delete the test case)

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Comment by soltysh** [comment]
> After we agree on the right approach how to proceed with partial errors, I've already added that topic to next sig-cli call upcoming Wed, we can continue from there in a separate PR.

@zhouya0 @apelisse so we talked about it during Wednesday's SIG-CLI and the overall agreement is that we'll want to see those partial errors, I'd suggesting something like `<error: %v>` if you want to open a PR.

**Comment by zhouya0** [comment]
> > After we agree on the right approach how to proceed with partial errors, I've already added that topic to next sig-cli call upcoming Wed, we can continue from there in a separate PR.
> 
> @zhouya0 @apelisse so we talked about it during Wednesday's SIG-CLI and the overall agreement is that we'll want to see those partial errors, I'd suggesting something like `<error: %v>` if you want to open a PR.

OK, I'll try to do this :) .

**Comment by apelisse** [comment]
Thanks @soltysh, that looks good!
